soccer/gameplay/situational_play_selection.py

```diff
@@ -298,60 +318,62 @@ def isMidfield(self):
     def isPileup(self):
         return self.currentPileup
 
-    ##Returns if we are in the specified situation without regard to capitalization
-    #If the check variable is true, it will check if the situation exists and throw an
-    #exception is it does now.
-    def isSituation(self, situation, check=False):
-        up = situation.upper()
-
-        if (check):
-            found = False
-            for g in self.Situation:
-                if (up == g):
-                    found = True
-                    break
-            if not found:
-                raise Exception("Passed situation " + situation + " / " + up +
-                                " is not an existing situation")
-
-        if (self.currentSituaion.name == up):
-            return True
+    ##Returns the current situation
+    def getSituation(self):
+        return self.currentSituation
+
+    ##
+    # 
+    # @param situation a situation as an enum
+    def isSituation(self, situation):
```

Comment:
I deprecated string support as it was confusing and adding complexity.

Reply:
:+1:, agree with this change.